TrackedObject[] blue_bricks
TrackedObject[] red_bricks
